src/theory/arith/nl/transcendental/transcendental_state.h

```diff
@@ -89,119 +89,120 @@ struct TranscendentalState
-    std::map<Node, Node> d_trMaster;
-    std::map<Node, std::unordered_set<Node, NodeHashFunction>> d_trSlaves;
-
-    /** concavity region for transcendental functions
-     *
-     * This stores an integer that identifies an interval in
-     * which the current model value for an argument of an
-     * application of a transcendental function resides.
-     *
-     * For exp( x ):
-     *   region #1 is -infty < x < infty
-     * For sin( x ):
-     *   region #0 is pi < x < infty (this is an invalid region)
-     *   region #1 is pi/2 < x <= pi
-     *   region #2 is 0 < x <= pi/2
-     *   region #3 is -pi/2 < x <= 0
-     *   region #4 is -pi < x <= -pi/2
-     *   region #5 is -infty < x <= -pi (this is an invalid region)
-     * All regions not listed above, as well as regions 0 and 5
-     * for SINE are "invalid". We only process applications
-     * of transcendental functions whose arguments have model
-     * values that reside in valid regions.
-     */
-    std::unordered_map<Node, int, NodeHashFunction> d_tf_region;
-    /**
-     * Maps representives of a congruence class to the members of that class.
-     *
-     * In detail, a congruence class is a set of terms of the form
-     *   { f(t1), ..., f(tn) }
-     * such that t1 = ... = tn in the current context. We choose an arbitrary
-     * term among these to be the repesentative of this congruence class.
-     *
-     * Moreover, notice we compute congruence classes only over terms that
-     * are transcendental function applications that are "master terms",
-     * see d_trMaster/d_trSlave.
-     */
-    std::map<Node, std::vector<Node>> d_funcCongClass;
-    /**
-     * A list of all functions for each kind in { EXPONENTIAL, SINE, POW, PI }
-     * that are representives of their congruence class.
-     */
-    std::map<Kind, std::vector<Node>> d_funcMap;
-
-    /** secant points (sorted list) for transcendental functions
-     *
-     * This is used for tangent plane refinements for
-     * transcendental functions. This is the set
-     * "get-previous-secant-points" in "Satisfiability
-     * Modulo Transcendental Functions via Incremental
-     * Linearization" by Cimatti et al., CADE 2017, for
-     * each transcendental function application. We store this set for each
-     * Taylor degree.
-     */
-    std::unordered_map<Node,
-                       std::map<unsigned, std::vector<Node>>,
-                       NodeHashFunction>
-        d_secant_points;
-
-    /** PI
-     *
-     * Note that PI is a (symbolic, non-constant) nullary operator. This is
-     * because its value cannot be computed exactly. We constraint PI to
-     * concrete lower and upper bounds stored in d_pi_bound below.
-     */
-    Node d_pi;
-    /** PI/2 */
-    Node d_pi_2;
-    /** -PI/2 */
-    Node d_pi_neg_2;
-    /** -PI */
-    Node d_pi_neg;
-    /** the concrete lower and upper bounds for PI */
-    Node d_pi_bound[2];
+                      int concavity);
+
+  Node d_true;
```

Comment:
May be a formatting issue here.

Reply:
Yes, but the new formatting is the right one.